libcodechecker/server/client_db_access_handler.py

```diff
@@ -11,6 +11,7 @@
 import codecs
 from collections import defaultdict
 import datetime
+from datetime import timedelta
```

Comment:
We only use `timedelta` at one place, I'd rather not have a separate import just for this.